backend/src/db/migrate/store.js

```diff
@@ -35,7 +35,7 @@ class Store {
     const session = driver.session()
     const readTxResultPromise = session.readTransaction(async txc => {
       const result = await txc.run(
-        'MATCH (migration:Migration) RETURN migration {.*} ORDER BY migration.timestamp DESC',
+        'MATCH (migration:Migration) RETURN migration {.*} ORDER BY migration.createdAt DESC',
```

Comment:
One could argue we should save two attributes: `createdAt` and `migratedAt`

Reply:
maybe, but I guess those will be the same cause the `Migration` is only created in the database when it is run for the first time... after that, since we use merge, it is not updated.
I thought about calling it `migratedAt`, and wouldn't be opposed to it, but thought that `createdAt` could also be ok...